src/test/java/com/squareup/javapoet/ParameterSpecTest.java

```diff
@@ -48,6 +101,28 @@
     }
   }
 
+  @Test public void fieldVariableElement() {
+    TypeElement classElement = getElement(VariableElementFieldClass.class);
+    List<VariableElement> methods = fieldsIn(elements.getAllMembers(classElement));
+    VariableElement element = findFirst(methods, "name");
+
+    try {
+      ParameterSpec.get(element);
```

Comment:
This tests will pass even if `get()` doesn't throw. This should be followed by `fail()`.